packages/diagrams/frontend/sirius-components-diagrams/src/renderer/layout/ListNodeLayoutHandler.ts

```diff
@@ -47,8 +47,13 @@ export class ListNodeLayoutHandler implements INodeLayoutHandler<ListNodeData> {
     forceDimensions?: ForcedDimensions
   ) {
     const nodeIndex = findNodeIndex(visibleNodes, node.id);
-    const nodeElement = document.getElementById(`${node.id}-listNode-${nodeIndex}`)?.children[0];
-    const borderWidth = nodeElement ? parseFloat(window.getComputedStyle(nodeElement).borderWidth) : 0;
+    const nodeElement = document.getElementById(`${node.id}-listNode-${nodeIndex}`);
+    const nodeElementChildren =
+      nodeElement?.children &&
+      Array.from(nodeElement.children).filter((child) => !child.classList.contains('react-flow__resize-control'))[0];
```

Comment:
Why filtering out the resize handle ?

Reply:
I have to filter it because when we get the first child (`document.getElementById(`${node.id}-listNode-${nodeIndex}`)?.children[0]` - before), it was the div containing the react flow resizer, so the size was not correct

Reply:
Okay I see, we should give to the needed div a dedicated `id `or `className `but it's out of the scope of this PR.